src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -754,7 +754,7 @@ class ZaakBesluitSerializer(NestedHyperlinkedModelSerializer):
 
     class Meta:
         model = ZaakBesluit
-        fields = ("url", "uuid", "besluit")
+        fields = ("url", "uuid", "besluit", "zaak")
```

Comment:
Without this change I was seeing the stack trace

```python
Traceback (most recent call last):
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 606, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1645, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 45, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1529, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1572, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "/Users/shea/Projects/open-zaak/src/openzaak/utils/admin.py", line 216, in save_model
    self.trail(obj, viewset, request, action, data_before, data)
  File "/Users/shea/Projects/open-zaak/src/openzaak/utils/admin.py", line 176, in trail
    main_object = data[viewset.audit.main_resource]
KeyError: 'zaak'
```

The last line `main_object = data[viewset.audit.main_resource]` tries to get `zaak` from `data` but there is no `zaak`.  This would make sure that `zaak` is returned in the data when this serializer is used.

Reply:
yeah adding this field is definitely not intended, as it breaks the [API spec](https://zaken-api.vng.cloud/api/v1/schema/)

Reply:
Makes sense, we're currently using an if/elif/else statement to get `main_object` so then it's probably better to just add another elif statement and undo this change.